Dockerfile

```diff
@@ -12,8 +12,9 @@ ENV DB_PORT 5432
 ENV DB_NAME pega
 
 # Provide variables for the JDBC connection string
-ENV JDBC_URL "jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}"
 ENV JDBC_CLASS org.postgresql.Driver
+ENV JDBC_DB_TYPE postgresql
```

Comment:
If not hard-coding postgres URL does it make sense to hard-code jdbc_class and db_type?